book/website/_toc.yml

```diff
@@ -326,6 +326,13 @@
       file: community-handbook/consistency/consistency-structure
     - title: Language
       file: community-handbook/consistency/consistency-language
+  - title: Contributing - Templates and Workflow
```

Comment:
```suggestion
  - title: Contributing and Developing Chapters - Templates and Workflow
```

Reply:
Just to be clear - case studies and subchapter are all part of "some" chapter. Without 'chapter' in the header, it doesn't fully make sense of what we mean by 'contributing'.